tests/functional/a/access/access_to_protected_members.py

```diff
@@ -85,9 +85,9 @@ def __le__(self, other):
         Test a correct access as the access to protected member
         is inside a special method even if it is deeply nested
         """
-        if 2 > 1:
+        if 2 > 1:  # pylint: disable=comparison-of-constants
```

Comment:
Generally we tend to disable at the file-level instead of in-line. We might want to keep that style here?

@Pierre-Sassoulas Do you have an opinion?

Reply:
I thought about this too, but I don't mind some tests for inline disabling sprinkled in our functional tests. This is something that is very hard to tests extensively if at some point we need to touch the disable code. Also not disabling everything permits to realize if there's issue with a change because our functional tests have very diverse code. A compromise has to be made between too much and zero, as it's also more work than just disabling per file. I think in this case it's ok.